pkg/workloadmeta/collectors/internal/containerd/containerd.go

```diff
@@ -271,7 +271,8 @@ func (c *collector) notifyInitialImageEvents(ctx context.Context, namespace stri
 
 	for _, image := range existingImages {
 		if err := c.notifyEventForImage(ctx, namespace, image, nil); err != nil {
-			return err
+			log.Warnf(err.Error())
```

Comment:
does the error is self explanatory? should add a small logs prefix to provide more context to the error?

Reply:
I wrapped the error to add some extra information :+1:
The wrapped error already contains everything that we need to debug the problem.